pkg/cache/v3/mux.go

```diff
@@ -37,6 +37,16 @@ type MuxCache struct {
 var _ Cache = &MuxCache{}
 
 func (mux *MuxCache) CreateWatch(request *Request, value chan Response) func() {
+	// Passing a Request by value to Classify triggers a govet copylocks
+	// error. This is because protobuf messages specifically embed
+	// a Mutex in order to trigger this check (i.e. not for actually
+	// locking fields).
+	//
+	// So in this specific case, it happens to be safe to copy the
```

Comment:
I would not be opposed to changing the API to not copy lock values. We did this for delta.

Reply:
OK, then I'll change this to take a pointer.